client/js/widgets/pile.js

```diff
@@ -208,15 +208,13 @@ class Pile extends Widget {
   }
 
   updateVisibleChildren() {
-    for (let child of this.visibleChildrenElements) {
+    for (const child of this.visibleChildrenElements) {
       child.classList.remove('visible-in-pile');
     }
     this.visibleChildrenElements.clear();
     let z = function (e) {
-      let zIndex = getComputedStyle(e).zIndex;
-      if (zIndex == 'auto')
-        return 0;
-      return zIndex;
+      // 'auto' should sort as 0.
```

Comment:
Not quite exactly, but close enough for this. Not sure that any widgets elements should every be 'auto'.

Function z never changes, so it can also be `const`